### ISSUE TITLE
Waterfox sold ?

### ISSUE THREAD

**Issue by ghost** [MrAlex94/Waterfox#1406]
Is the end of Waterfox as we know it ?? [](url)https://www.ghacks.net/2020/02/14/waterfox-web-browser-sold-to-system1/

**Comment by hawkeye116477** [comment]
It's rather not good place for discussion, so I refer to https://www.reddit.com/r/waterfox/comments/f3hi8s/privacy_browser_waterfox_appears_to_be_sold_to/fhistpt/.

**Comment by linuxgirl22** [comment]
Appreciate the reddit link, but, as I don't use reddit, I hope it's okay just to add a brief comment into the thread. So disappointed, and agree it's the end of waterfox ... the majority of people came to waterfox for privacy, and the last thing wanted was linking with bing/system1 etc (e.g. I just stopped using ddg because of system1). Without users, the funding obtained will probably leave at some point in the future. Kickstarters/asking more directly for donations seems to have skipped over, but, anyway, thanks for the period of time where waterfox was private. Appreciated that.

**Comment by smaragdus** [comment]
@linuxgirl22 

> the majority of people came to waterfox for privacy

[Waterfox](https://www.waterfox.net/) and privacy do not stick together- have a look at this [article](https://spyware.neocities.org/articles/waterfox.html).

**Comment by hawkeye116477** [comment]
Stay calm and look at MrAlex's replies on reddit. Anyway, for privacy probably Tor would be better, Waterfox's strength is rather customizability.

`Waterfox and privacy do not stick together- have a look at this article.`
That was mentioned previously and that site is full of fake news. See https://github.com/MrAlex94/Waterfox/issues/1264#issuecomment-553531702.

**Comment by smaragdus** [comment]
@hawkeye116477 

> Stay calm and look at MrAlex's replies on reddit.

I am calm, actually I don't use [Waterfox](https://www.waterfox.net/)- I tested it once, I didn't like it and I removed it.

> That was mentioned previously and that site is full of fake news. See https://github.com/MrAlex94/Waterfox/issues/1264#issuecomment-553531702.

Who says this [site](https://spyware.neocities.org/) is full of fake news? You?

Anyway, I don't trust any browser, even the Chromium fork I am currently using. Yet I consider a couple of browsers- [Iridium](https://iridiumbrowser.de/) and [Pale Moon](https://www.palemoon.org/), to be a bit more trustworthy than the others.

**Comment by MrAlex94** [comment]
This has literally nothing to do with Waterfox issues. Displays like this are not okay.

> the majority of people came to waterfox for privacy

That’s literally not true. Waterfox has never dressed itself up as a privacy tool. Nothing has changed in that regard but honestly I can’t seem to change how people keep making things up.

Also that NeoCities article has been debunked so many times, it takes away from what you’re trying to say.

**Comment by smaragdus** [comment]
@MrAlex94 

> This has literally nothing to do with Waterfox issues.

Personally I am not concerned because currently I do not have [Waterfox](https://www.waterfox.net/). Yet I believe that users have the right to be informed about sensitive privacy stuff like this one. 

> Also that NeoCities article has been debunked so many times, it takes away from what you’re trying to say.

The NeoCities [website](https://spyware.neocities.org/) has [articles](https://spyware.neocities.org/articles/index.html) about most major browsers, not just about [Waterfox](https://spyware.neocities.org/articles/waterfox.html) and these series of articles expose privacy issues with most browsers that have been reviewed. The fact that you say that the article has been debunked does not mean that it really has, in order to be debunked it needs proof to be provided, not just some random weasel talk.

For those who seek for alternatives they might check [Otter](https://otter-browser.org/) browser which seems to be [clean](https://spyware.neocities.org/articles/otter.html).

**Comment by MrAlex94** [comment]
> Yet I believe that users have the right to be informed about sensitive privacy stuff like this one.

But nothing has changed...? No offense but you're being paranoid. It's not a way to live life.

**Comment by MrAlex94** [comment]
@smaragdus 
> 
> > Stay calm and look at MrAlex's replies on reddit.
> 
> I am calm, actually I don't use [Waterfox](https://www.waterfox.net/)- I tested it once, I didn't like it and I removed it.
> 
> > That was mentioned previously and that site is full of fake news. See [#1264 (comment)](https://github.com/MrAlex94/Waterfox/issues/1264#issuecomment-553531702).
> 
> Who says this [site](https://spyware.neocities.org/) is full of fake news? You?
> 
> Anyway, I don't trust any browser, even the Chromium fork I am currently using. Yet I consider a couple of browsers- [Iridium](https://iridiumbrowser.de/) and [Pale Moon](https://www.palemoon.org/), to be a bit more trustworthy than the others.

Sorry but that NeoCities website has been debunked plenty of times. Posting it over and over doesn't change that. Anyone rational can sit and go through it and verify the "claims" themselves.

Secondly, did Pale Moon not have [Start.me as default tab page](https://palemoon.start.me/start)? Like, have you read the [privacy policy](https://start.me/privacy)?

> Among the types of Personal Data that start.me collects, by itself or through third parties, there are: Cookies; Usage Data; first name; last name; email address; phone number.

But of course Waterfox, which collects absolutely nothing from its users is the problem.

**Comment by linuxgirl22** [comment]
"the majority of people came to waterfox for privacy"
That’s literally not true. Waterfox has never dressed itself up as a privacy tool. Nothing has changed in that regard but honestly I can’t seem to change how people keep making things up.
Also that NeoCities article has been debunked so many times, it takes away from what you’re trying to say.

Waterfox was advertised as firefox without telemetry etc, so those wanting privacy would cross over to it. Waterfox has changed it's focus, clearly, but for most of the time I was using it, everyone was referencing being glad to have the privacy features. And I didn't post the Neocities article.

**Comment by Ibuprophen** [comment]
I heard that Global Warming is directly the result of those who use Waterfox... LMAO! 

I'm surprised that someone like @MrAlex94 didn't lock this conversation/issue for being off topic. 


~Ibuprophen 
[Theoretical Theorist]

**Comment by smaragdus** [comment]
@MrAlex94 

> But nothing has changed...? No offense but you're being paranoid. It's not a way to live life.

I agree with you about this- when it comes to browsers and WWW I am really paranoid as I witnessed how the web has turned into one giant adware and spyware machine. The future of web browsing looks very dreary to me and I will have to sacrifice convenience in order to retain a small portion of privacy (which may be only imaginable). I have followed the evolution of both Firefox and Opera for example and it is very disturbing (disturbing is the mildest term I could use).

> Sorry but that NeoCities website has been debunked plenty of times. Posting it over and over doesn't change that. Anyone rational can sit and go through it and verify the "claims" themselves.

The best way to assess whether the claims are real or not is to set up a network analyzer ([Wireshark](https://www.wireshark.org/) for example) and check a freshly installed browser for background requests. For the time being I don't have enough time for such a task. As [Waterfox](https://www.waterfox.net/) is based on [Firefox](https://www.mozilla.org/en-US/firefox/) which is not a lesser privacy disaster than [Chromium](https://www.chromium.org/Home) it is logical to suppose that it abuses the users the same way [Firefox](https://www.mozilla.org/en-US/firefox/) does. And when one discovers that a browser has been acquired by the same company which acquired [Startpage](https://www.startpage.com/) one cannot help but become highly suspicious or paranoid- [System1](https://www.system1.com/) is not a charity organization after all but an advertising company.

> Secondly, did Pale Moon not have [Start.me as default tab page](https://palemoon.start.me/start)? Like, have you read the [privacy policy](https://start.me/privacy)?

I have [Pale Moon](https://www.palemoon.org/) as a reserve browser and I use it quite rarely. I have set it up long ago yet I remember that it used some very ugly start page. When I install a browser the first thing I do is attempting to clean it up a bit by changing the new tab page,  the default search engine and disabling Google services.

> But of course Waterfox, which collects absolutely nothing from its users is the problem.

I really do not know what [Waterfox](https://www.waterfox.net/) collects or sends as I have not tried to check this myself. I happened to come here by chance- I am not a user of your browser, and I don't think I will come back. It is just that I am concerned with all browsers and their malicious practices of tracking the users and selling the harvested user data,

**Comment by grahamperrin** [comment]
> I don't use Waterfox

– and: 

> I have not tried

Hm

**Comment by ghost** [comment]
Time will tell if @MrAlex94 is writing the truth.

**Comment by grahamperrin** [comment]
> if @MrAlex94 is writing the truth.

How rude.

**Comment by ghost** [comment]
@grahamperrin 
where do you see rudeness. ??

**Comment by smnthermes** [comment]
> Sorry but that NeoCities website has been debunked plenty of times. 

Neocities is a host. So by your reasoning what you're saying is invalid because GitHub has been debunked plenty of times. 😂

[Edited for more clarity]

**Comment by grahamperrin** [comment]
> Neocities is a host. … more clarity]

The reference was to a specific site. Not to https://neocities.org/

**Comment by jorg35** [comment]
And someone was cheating here >> [](url)https://www.waterfox.net/blog/waterfox-has-joined-system1/

**Comment by grahamperrin** [comment]
> jorg35 added the bug label 

This was not a bug. 

@jorg35 wrote: 

> And someone was cheating here >> https://www.waterfox.net/blog/waterfox-has-joined-system1/

https://redd.it/f3zm8o discusses the blog post.

**Comment by dimyme** [comment]
>   [Waterfox](https://spyware.neocities.org/articles/waterfox.html) and these series of articles expose privacy issues with most browsers that have been reviewed. The fact that you say that the article has been debunked does not mean that it really has, in order to be debunked it needs proof to be provided, not just some random weasel talk. 

Indeed! There are tremendous amounts of weasel words here, that is very disturbing to say the least.

**Comment by dimyme** [comment]
> But of course Waterfox, which collects absolutely nothing from its users is the problem.

Would it be any better if instead Mozilla, enabled by waterfox, collects something from its users? Sounds a bit weaselish to me...

**Comment by grahamperrin** [comment]
@LeeBinder you wonder about the inexplicable voting at <https://github.com/MrAlex94/Waterfox/issues/582#issuecomment-596269842>; you might wonder about votes elsewhere. 

https://github.com/MrAlex94/Waterfox/issues/1406#issuecomment-586561327 above might help to set the tone.

**Comment by grahamperrin** [comment]
@smaragdus [wrote](https://github.com/MrAlex94/Waterfox/issues/1406#issuecomment-586277655): 

> … privacy … this article.

### The self-described 'Spyware Watchdog' and its filter bubble

`https://spyware.neocities.org/articles/waterfox.html` and other articles.

_Spyware Watchdog_ articles are thoroughly disreputable, and saying so is not Waterfox project bias. 

### Reputable advice

Consider the words of **a moderator in the /r/privacy subreddit** – without reference to Waterfox – pinned (sticky), emphasising the unreliable nature of the so-the called _Spyware Watchdog_ articles: 

> … rules:
> 
>> Please don’t fuel conspiracy thinking here. Don’t try to spread FUD, especially against reliable privacy-enhancing software. Extraordinary claims require extraordinary evidence. Show credible sources.
> 
> … please use better, more reliable sources. Thank you.

– and: 

> The neocities sites OP links to have been picked apart on this subreddit at length many times over. As mod, I don't have the time to get into it every time someone links to them. I can warn, which may cause people to ask why, and yet others can answer them. Mods live by the same restrictions of time and space as everyone else. We can't do everything :)

– https://old.reddit.com/r/privacy/comments/epmybg/privacy_is_already_dead_why_your_tech_solutions/felcy2m/?context=2

**Comment by LeeBinder** [comment]
> @LeeBinder you wonder about the inexplicable voting at [#582 (comment)](https://github.com/MrAlex94/Waterfox/issues/582#issuecomment-596269842); you might wonder about votes elsewhere.
> 
> [#1406 (comment)](https://github.com/MrAlex94/Waterfox/issues/1406#issuecomment-586561327) above might help to set the tone.

LOL no you misinterpreted.. jorg35 downvoted [YOUR post](https://github.com/MrAlex94/Waterfox/issues/582#issuecomment-596212830) not mine ..

**Comment by smnthermes** [comment]
> Reputable advice

> Consider the words of a moderator in the /r/privacy subreddit

Who says that a mod in r/privacy is reputable? You?! SMH.

I'd say they aren't reputable, based on [their attitude described there](https://old.reddit.com/r/privatelife/comments/f26qsc/threat_models_indoctrination_bias_and_criticism/).

**Comment by smnthermes** [comment]
P.S.: I wonder if @grahamperrin has become a PR employee for System1, by looking at [this post](https://old.reddit.com/r/privacy/comments/epmybg/privacy_is_already_dead_why_your_tech_solutions/fj7qhlo/).

Edit: [System1 definitely doesn't have ethics.](http://techrights.org/2020/01/23/relisting-for-money/)

**Comment by grahamperrin** [comment]
> I wonder if @grahamperrin has become a PR employee for System1,

I'm flattered. 

As I wrote last week, the _Community Support_ role (in the [Waterfox support area](https://www.reddit.com/r/waterfox/)) is entirely voluntary, not under anyone's direction.

**Comment by MrAlex94** [comment]
Oh dear, let's settle this once and for all. 

First off, lets start with the most egregious thing. In the "[Top Tier Privacy](https://spyware.neocities.org/articles/browsers.html)" list, we have Pale Moon. Now, Pale Moon has https://palemoon.start.me/start (https://github.com/MoonchildProductions/Pale-Moon/blob/master/palemoon/branding/shared/locales/browserconfig.properties) as the default the home page. Let's check out the [privacy policy](https://palemoon.start.me/privacy):

>  Among the types of Personal Data that start.me collects, by itself or through third parties, there are: Cookies; Usage Data; first name; last name; email address; phone number. 

There is literally no privacy there. Why is it at the top of the list? Waterfox literally collects none of that data. Nothing as personal as Names or Phone numbers - literally the opposite of private. In fact, the "[review](https://spyware.neocities.org/articles/palemoon.html)" for Pale Moon doesn't even mention this page, how weird considering the above policy.

Next, the Waterfox article:

> If you start up Waterfox for the first time, it will make 109 requests.

Unfortunately it's the language packs checking for updates to them. This will be remedied. They point to non-existent URLs, so the requests don't do anything. Current doesn't have this problem.

> Waterfox has a communication problem

Debatable. The author of the article doesn't seem to understand to how read and process a privacy policy, as you'll see below.

Not only that, but there is a complaint there's no description of what is sent to Google SafeBrowsing...which is why I linked the privacy policies that the author complains about.

> Waterfox "phones home" with information about your computer whenever you start it up
> Waterfox version
> Operating system
> Language settings
> Installed Waterfox Add-ons

Tor *literally* does the same thing when it starts up:
https://gitweb.torproject.org/tor-browser.git/tree/browser/app/profile/firefox.js?h=tor-browser-68.6.0esr-9.5-1-build1#n159

See that? Tor version, OS and language. If you have add-ons installed, they check for updates as well. Just like Waterfox. The behaviour is exactly the same. 

> Waterfox sends all website notifications through Mozilla's servers

That is literally how the web standard works. It only does that if you use Web Notifications and Mozilla are pretty on point with the privacy of their web servers. Don't use web notification? Not a problem then. That's why it's in the privacy policy.

> Waterfox is integrated into the "Firefox Accounts" spyware platform

I mean seriously, people complain if this feature isn't available. Because Waterfox is privacy-conscious, not a privacy tool like Tor which has stripped these features out. 

> Waterfox is self updating software.

So is Tor. [It literally self updates as well](https://gitweb.torproject.org/tor-browser.git/tree/browser/app/profile/firefox.js?h=tor-browser-68.6.0esr-9.5-1-build1#n147).

Next problem, the authors definition of spyware is literally incorrect.

> Software that enables a user to obtain covert information about another's computer activities by transmitting data covertly from their hard drive.

Literally nothing covert is happening, since it's out in the open. 

So to summarise:

Waterfox and Tor behave the same in terms of what "data" is "sent back" to their respective servers (add-ons checking for updates, OS type, language and browser version).

A valid criticism is the amount of connections done for the language packs.

Pale Moon's start.me service - if used - collect Name, phone, email. Opposite of private but is at the top.

I think I have thoroughly debunked the article now. Almost all the things that Waterfox apparently does, are also done by Tor.

**Comment by grahamperrin** [comment]
> System1 definitely doesn't have ethics.

### Techrights

`http://techrights.org/2020/01/23/relisting-for-money/` and other articles. 

Another thoroughly disreputable site, and saying so is not Waterfox project bias.

The quote at https://framasphere.org/posts/c8b66200339801381e6d002590d8e506#9881373042d00138e63c2a0000053625 is from Dan Arel – [columnist, author, and curator of `www.thinkprivacy.io`](https://forum.privacytools.io/u/danarel).

Here, the original: 

![image](https://user-images.githubusercontent.com/192271/76474784-961eea80-63f4-11ea-9953-d13b58decab2.png)

----

Elsewhere, the ranting of the guy behind Techrights is described as _insane_ and a moderator observes that he was banned for spamming: https://old.reddit.com/r/linux/comments/f4yaym/waterfox_has_joined_system1/fi2gdwx/?context=2

**Comment by grahamperrin** [comment]
> … jorg35 downvoted YOUR post not mine ..

Thanks @LeeBinder I didn't notice a down-vote against yours, but I did see the spamming by @jorg35. 

Re: https://github.com/MrAlex94/Waterfox/issues/1406#issuecomment-586561327 above, maybe someone can explain the rudeness of asking questions then inferring that the answers are lies.
